docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -44,7 +44,10 @@ See [Troubleshooting](doc:troubleshooting#how-to-change-your-cache-directory) fo
 >    size_mb=$(du -m -d0 "${path}" | cut -f 1)
 >    if (( size_mb > limit_mb )); then
 >      echo "${path} is too large (${size_mb}mb), nuking it."
->      rm -rf "${path}"
+>      parent_dir=$(dirname "${path}")
+>      nuke_path=$(mktemp -d "${parent_dir}/nuke.XXXXXX")
```

Comment:
This is fine since Pants is essentially perfectly unlikely to use a cache dir named nuke, but using `nuke_path="$(mktemp -d "${parent_dir}/$(basename "${path}").XXXXXX")"` with a corresponding `rm -rf` adjustment seems prudent.